src/main/io/rcsplit.c

```diff
@@ -82,7 +82,11 @@ static void rcSplitProcessMode(void)
             uint8_t argument = RCSPLIT_CTRL_ARGU_INVALID;
             switch (i) {
             case BOXCAMERA1:
-                argument = RCSPLIT_CTRL_ARGU_WIFI_BTN;
+                // check  whether arm unlock, we found a bug in rcsplit firmware:
+                // if rcsplit running without Wi-Fi module, and user try to turn on wifi, it'll cause rcsplit to turn off itself, this is danger
+                if (!ARMING_FLAG(ARMED)) {
```

Comment:
This will need to include `fc/runtime_config.h` to build without a warning.

Reply:
OK! I get it, I've fix the complie error, and rebase the PR